db/schema.rb

```diff
@@ -28,8 +28,8 @@
   create_sequence "candidates_id_seq"
   create_sequence "chasers_sent_id_seq"
   create_sequence "course_options_id_seq"
-  create_sequence "course_subjects_id_seq"
   create_sequence "courses_id_seq"
+  create_sequence "course_subjects_id_seq"
```

Comment:
could we also avoid committing the moved around sequences and only keep the removal of the column